packages/microcosm/test/unit/observable.test.js

```diff
@@ -309,4 +309,62 @@ describe('Observable', function() {
       )
     })
   })
+
+  describe('::filter', () => {
+    it('operates like array.filter over a stream of values', () => {
+      let isOdd = n => n % 2
+      let double = Observable.of(0, 1, 2, 3, 4).filter(isOdd)
+      let answers = []
+
+      double.subscribe(value => answers.push(value))
+
+      scheduler().flush()
```

Comment:
is `double` `[1, 3]` before the subscribe line? is it worth asserting that `answers` is empty until after the scheduler has been flushed?

Reply:
It does not because it is lazy. The sequence doesn't process until you call `subscribe`. 

Seems like a good clarification anyway though, and a nice confirmation should I refactor this later. I'll add it.

Reply:
Did this in a5ee6303